src/main/java/at/hannibal2/skyhanni/features/slayer/blaze/FirePillarDisplay.kt

```diff
@@ -0,0 +1,44 @@
+package at.hannibal2.skyhanni.features.slayer.blaze
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.IslandType
+import at.hannibal2.skyhanni.events.GuiRenderEvent
+import at.hannibal2.skyhanni.events.LorenzTickEvent
+import at.hannibal2.skyhanni.utils.EntityUtils
+import at.hannibal2.skyhanni.utils.LorenzUtils.isInIsland
+import at.hannibal2.skyhanni.utils.RenderUtils.renderString
+import at.hannibal2.skyhanni.utils.StringUtils.matchFirst
+import net.minecraft.entity.item.EntityArmorStand
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class FirePillarDisplay {
+    private val config get() = SkyHanniMod.feature.slayer.blazes
+
+    private var display = ""
+
+    @SubscribeEvent
+    fun onTick(event: LorenzTickEvent) {
+        if (!isEnabled()) return
+
+        val pattern = "§6§l(?<seconds>.*)s §c§l8 hits".toPattern()
```

Comment:
is there a reason to not repopattern this?

Reply:
forgot. ty, will fix.